test/App.test.js

```diff
@@ -7,3 +7,4 @@ it('renders without crashing', () => {
       .toEqual(1)
       .toBeGreaterThan(0);
```

Comment:
What is this file Chris? Unit test?

Reply:
Oh yeah, I was going to get some unit testing working too. Haven't got round to it yet.